index.html

```diff
@@ -5,8 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Henry Cadena Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Henry Cadena Portfolio Mobile First</h1>
+    <header class="container-header">
+      <h3 class="logo">Henry Cadena</h3>
```

Comment:
- [ ] For convention, semantics, and a better user experience (UX), it is advisable to place the logo inside a `<a> `tag. 🤔🤔
- [ ] There are three main reasons for this choice. First, it follows the conventional approach where logos are often wrapped in anchor tags to indicate that they are clickable.
- [ ]  Second, from a semantic standpoint, placing the logo within a `<a>` tag signifies that it represents a link, improving accessibility and search engine optimization. 
- [ ] Lastly, incorporating the logo within an anchor tag enhances the user experience by providing a consistent and expected behavior, allowing users to navigate to the homepage or related content by clicking on the logo.
- [ ] For more insights [click here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)💭💭🤔

Reply:
Thanks @AbdusSattar-70